scripts/jenkins/qa_openstack.sh

```diff
@@ -365,7 +365,15 @@ outputs:
   server_floating_ip:
     value: { get_attr: [ my_floating_ip, floating_ip_address ] }
 EOF
+
+os_client_ret=1
+
 if [ -x /usr/bin/openstack ]; then
+    openstack stack list > /dev/null 2>&1
```

Comment:
This check is sufficient. It is also possible to check `openstack --version`. I will adopt changed